src/pretix/plugins/paypal/payment.py

```diff
@@ -134,6 +148,68 @@ def settings_form_fields(self):
                  )),
             ]
 
+        methods = [
+            ('method_wallet',
+             forms.BooleanField(
+                 label=_('PayPal'),
+                 required=False,
+                 help_text=_(
+                     'Even if a customer choses an Alternative Payment Method, they will always have the option to '
+                     'revert back to paying with their PayPal account. For this reason, this payment method is always '
+                     'active.'
+                 ),
+                 disabled=True,
+             )),
+            ('method_apm',
+             forms.BooleanField(
+                 label=_('Alternative Payment Methods'),
+                 help_text=_(
+                     'In addition to payments through a PayPal account, you can also offer your customers the option '
+                     'to pay with credit cards and other, local payment methods such as SOFORT, giropay, iDEAL, and '
+                     'many more - even when they do not have a PayPal account. Eligible payment methods will be '
+                     'determined based on the shoppers location. For German merchants, this is the direct successor '
+                     'of PayPal Plus.'
+                 ),
+                 required=False,
+                 widget=forms.CheckboxInput(
+                     attrs={
+                         'data-checkbox-dependency': '#id_payment_paypal_method_wallet',
+                     }
+                 )
+             )),
+            ('disable_method_sepa',
+             forms.BooleanField(
+                 label=_('Disable SEPA Direct Debit'),
+                 help_text=_(
+                     'While most payment methods cannot be recalled by a customer without outlining their exact grief '
+                     'with the merchants, SEPA Direct Debit can be recalled with the press of a button. For that '
+                     'reason - and depending on the nature of your event - you might want to disabled the option of '
+                     'SEPA Direct Debit payments in order to reduce the risk of costly chargebacks.'
+                 ),
+                 required=False,
+                 widget=forms.CheckboxInput(
+                     attrs={
+                         'data-checkbox-dependency': '#id_payment_paypal_method_apm',
+                     }
+                 )
+             )),
+            ('enable_method_paylater',
+             forms.BooleanField(
+                 label=_('Enable Buy Now Pay Later'),
+                 help_text=_(
+                     'Offer your customers the possibility to buy now (up to 1.000€) and pay in 3/6/12/24 installments '
```

Comment:
```suggestion
                     'Offer your customers the possibility to buy now (up to a certain limit) and pay in multiple installments '
```
Let's maybe not hardcode things that are likely country/currency-specific or could be changed by paypal any day?